src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_evp.h

```diff
@@ -120,3 +120,12 @@ GetMaxMdSize
 Returns the maxium bytes for a message digest.
 */
 PALEXPORT int32_t CryptoNative_GetMaxMdSize(void);
+
+PALEXPORT int32_t CryptoNative_Pkcs5Pbkdf2Hmac(const char* password,
```

Comment:
In the original shim we used names like this, because we were just directly shimming the native functions.

In newer operations I've gone for more naming the operation whatever I felt was appropriate.  I'd call this `CryptoNative_Pbkdf2`, for example.